ui/raidboss/data/07-dt/raid/r7s.txt

```diff
@@ -1,9 +1,323 @@
-### Arcadion (R7S): AAC Cruiserweight M3 Savage
+### AAC CRUISERWEIGHT M3 (SAVAGE)
+# ZoneId: 1261
+
+# -ii A563 A564 A574 A577 A578 A585 A589 A58C A58D A58F A591 A594 A595 A5A2 A5A4 A5A6 A5A7 A5A9 A5AB A5AC A5AD A5B2 A63C A63D A94A
+# -it "Blooming Abomination" "Brute Abombinator"
 
 hideall "--Reset--"
 hideall "--sync--"
 
 0.0 "--Reset--" ActorControl { command: "4000000F" } window 0,100000 jump 0
 
 0.0 "--sync--" InCombat { inGameCombat: "1" } window 0,1
+3.4 "--sync--" Ability { id: "A55A", source: "Brute Abombinator" }
+10.5 "Brutal Impact x6" Ability { id: "A55B", source: "Brute Abombinator" } duration 5.2
+18.1 "--sync--" Ability { id: "A55A", source: "Brute Abombinator" }
+23.1 "Stoneringer" Ability { id: ["A55D", "A55E"], source: "Brute Abombinator" }
+32.0 "Smash Here/Smash There" Ability { id: ["A55F", "A560"], source: "Brute Abombinator" }
+33.0 "Brutish Swing" Ability { id: ["A561", "A562"], source: "Brute Abombinator" }
+34.1 "Brutal Smash" Ability { id: ["A565", "A566"], source: "Brute Abombinator" }
+36.1 "--sync--" Ability { id: "A55A", source: "Brute Abombinator" }
+39.1 "--sync--" Ability { id: "A55A", source: "Brute Abombinator" }
+43.1 "Spore Sac (cast)" Ability { id: "A569", source: "Brute Abombinator" }
+46.2 "--sync--" Ability { id: "A516", source: "Brute Abombinator" }
+48.2 "Spore Sac" Ability { id: "A56A", source: "Brute Abombinator" }
+52.6 "Sinister Seeds (cast)" Ability { id: "A56D", source: "Brute Abombinator" }
+53.7 "Pollen" Ability { id: "A56B", source: "Brute Abombinator" }
+55.8 "--adds targetable--"
+56.6 "Sinister Seeds (puddles 1)" #Ability { id: "A571", source: "Brute Abombinator" }
+58.6 "Sinister Seeds (puddles 2)" #Ability { id: "A571", source: "Brute Abombinator" }
+60.6 "Sinister Seeds (seeds drop)" Ability { id: "A56E", source: "Brute Abombinator" }
+60.6 "Sinister Seeds (puddles 3)" #Ability { id: "A571", source: "Brute Abombinator" }
+62.6 "Sinister Seeds (puddles 4)" #Ability { id: "A571", source: "Brute Abombinator" }
+65.2 "Impact" Ability { id: "A573", source: "Brute Abombinator" }
+65.2 "Tendrils of Terror" Ability { id: ["A56F", "A570"], source: "Brute Abombinator" }
+70.5 "Roots of Evil" Ability { id: "A572", source: "Brute Abombinator" }
+75.2 "--sync--" Ability { id: "A55A", source: "Brute Abombinator" }
+78.2 "--sync--" Ability { id: "A55A", source: "Brute Abombinator" }
+78.2 "Winding Wildwinds (adds)" #Ability { id: "A90D", source: "Blooming Abomination" }
+78.2 "Crossing Crosswinds (adds)" #Ability { id: "A90E", source: "Blooming Abomination" }
```

Comment:
```suggestion
78.2 "Winding Wildwinds/Crossing Crosswinds (adds)" #Ability { id: ["A90D", "A90E"], source: "Blooming Abomination" }
```
No strong feelings on this, but visually there's not a lot of use for separating these in the timeline. Maybe combine it with a replacement to show it as `Crosswinds`?